.eslintrc.yml

```diff
@@ -253,4 +249,5 @@ rules:
   react/self-closing-comp: 1
   react/sort-comp: 1
   react/sort-prop-types: 1
-  react/jsx-wrap-multilines: 1
+
+  prettier/prettier: 2
```

Comment:
These red 🚫 marks from GH always stand out to me. Even though there's no practical consequence for us not having these in C# land, can we please do it for consistency and to keep the warning-looking-things out of our PRs?

Reply:
Good point. I've modified eol-last to error, and updated vscode settings to add a new line on save, automatically.